Makefile

```diff
@@ -86,6 +100,10 @@ set_env_class_prod:
 	$(eval export AWS_ACCOUNT=prod)
 	$(eval export ENABLE_AUTO_DEPLOY=true)
 	$(eval export PAAS_CF_TAG_FILTER=prod-*)
+	$(eval export ROOT_DNS_ZONE=cloud.service.gov.uk)
+	$(eval export APPS_ROOT_DNS_ZONE=cloudapps.digital)
+	$(eval export ENV_DNS_ZONE=${AWS_ACCOUNT}.${ROOT_DNS_ZONE})
```

Comment:
We shouldn't be including `AWS_ACCOUNT` in the domains for production, they should be using the root domains directly.